integration_test/sql/migration.exs

```diff
@@ -71,6 +71,43 @@ defmodule Ecto.Integration.MigrationTest do
     end
   end
 
+  defmodule AlterColumnFromMigration do
+    use Ecto.Migration
+
+    def change do
+      create table(:modify_from_authors, primary_key: false) do
+        add :id, :integer, primary_key: true
+      end
+      create table(:modify_from_posts) do
+        add :author, references(:modify_from_authors, type: :integer)
+      end
+
+      if direction() == :up do
+        execute "INSERT INTO modify_from_authors (id) VALUES (1)"
```

Comment:
Let's use `Repo.insert_all "modify_from_authors", [[id: 1]]` and `Repo.insert_all "modify_from_posts", [[author: 1]]`. I think you will have to call `flush()` before though, to ensure the migration is executed before.